### ISSUE TITLE
Consider removing plot_roc_curve

### ISSUE THREAD

**Issue by kaqmak** [dfds-data/dfds-ds-toolbox#31]
Does the `plot_roc_curve` provide anything addtional than what is in Scikit-learn? 
See [RocCurveDisplay](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.RocCurveDisplay.html#sklearn.metrics.RocCurveDisplay)

From what I see, it does not.

**Comment by KPLauritzen** [comment]
Agreed. We have been discussing if there is any value in having a consistent interface (or just a list of common plots) here. Most of the plots we do are already provided somewhere else.

Especially yellowbrick https://www.scikit-yb.org/en/latest/ has a huge overlap with the plots we are doing.

**Comment by nybyhansen** [comment]
What do you feel @dfds-data/data-science-chapter? Should we remove this plot or let it be for now? 💀 

As @KPLauritzen mentions, it could be valuable if we had the same interface for all of our plots. That way we would be able to plug them into a training component that pushes the plot artifacts to `mlflow` (as an example).

**Comment by KPLauritzen** [comment]
I like the idea of having one interface for all the plots. 

However, if we want this library to be installed in many projects, we will suddenly drag in many extra dependencies. Maybe we can add yellowbrick or seaborn or some other plotting library as an optional dependency?
